package.json

```diff
@@ -5,8 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"No tests specified\" && exit 0",
-    "start": "node server.js",
-    "dev": "nodemon server.js"
```

Comment:
we need this for the remote deployed server, you can still use the dev same as what start is doing.